tradingstrategy/exchange.py

```diff
@@ -19,6 +19,44 @@
 from tradingstrategy.types import NonChecksummedAddress, UNIXTimestamp, PrimaryKey
 
 
+class ExchangeNotFoundError(Exception):
```

Comment:
Can you make these exceptions to have the same parent class base exception so they can be easily caught?